### PR TITLE
Update to spec v0.6.1

### PR DESCRIPTION
**Pull request by michaelsproul**
## Proposed Changes

Update to v0.6.1 of the spec

## Additional Info

So far I've done most of per-epoch processing (up to registry updates).

### CODE REVIEW

File: eth2/types/src/indexed_attestation.rs [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    CachedTreeHash,
+    TestRandom,
+    SignedRoot,
+)]
+pub struct IndexedAttestation {
+    /// Lists validator registry indices, not committee indices.
+    pub custody_bit_0_indices: Vec<u64>,
+    pub custody_bit_1_indices: Vec<u64>,
+    pub data: AttestationData,
+    pub custody_bitfield: Bitfield,
```
Comment (paulhauner) [inline eth2/types/src/indexed_attestation.rs]: I think `custody_bitfield` doesn't belong here anymore :)

https://github.com/ethereum/eth2.0-specs/blob/v0.6.1/specs/core/0_beacon-chain.md#indexedattestation

Comment (paulhauner) [reply eth2/types/src/indexed_attestation.rs]: Additionally, I noticed we're missing the graffiti field on `BeaconBlockBody`. I couldn't find the code in this diff.

**Comment by paulhauner** [comment]
Hey, I started working on this in #380 because I was having issues pushing to this branch (it's on your local repo) :)

**Comment by michaelsproul** [comment]
Closing in favour of #380

**Pull request closed without merging**